peekingduck/pipeline/nodes/heuristic/bbox_to_btm_midpoint.py

```diff
@@ -16,14 +16,27 @@
 Converts bounding boxes to a single point of reference
 """
 
-
 from typing import Dict, Any, Tuple
 from peekingduck.pipeline.nodes.node import AbstractNode
 
 
 class Node(AbstractNode):
-    """This node converts bounding boxes to a singel pt which is
-    the bottom midpoint of the bounding box"""
+    """This node converts bounding boxes to a single point which is
+    the bottom midpoint of the bounding box
+
```

Comment:
can give a short explanation on what this is typically used for? i.e. zone counting

Reply:
Added